clang/include/clang/CIR/Interfaces/ASTAttrInterfaces.td

```diff
@@ -63,20 +63,48 @@ let cppNamespace = "::cir" in {
 
   def ASTVarDeclInterface : AttrInterface<"ASTVarDeclInterface",
       [ASTDeclaratorDeclInterface]> {
-    let methods = [
-      InterfaceMethod<"", "void", "mangleDynamicInitializer", (ins "llvm::raw_ostream&":$Out), [{}],
-        /*defaultImplementation=*/ [{
+    let methods = [InterfaceMethod<"", "void", "mangleDynamicInitializer",
+                                   (ins "llvm::raw_ostream&":$Out), [{}],
+                                   /*defaultImplementation=*/[{
           std::unique_ptr<clang::MangleContext> MangleCtx(
             $_attr.getAst()->getASTContext().createMangleContext());
             MangleCtx->mangleDynamicInitializer($_attr.getAst(), Out);
-          }]
-      >,
-      InterfaceMethod<"", "clang::VarDecl::TLSKind", "getTLSKind", (ins), [{}],
-        /*defaultImplementation=*/ [{
+          }]>,
+                   InterfaceMethod<"", "void", "mangleStaticGuardVariable",
```

Comment:
Indenting in this file is pretty weird, doesn't match the rest of the file - please fix this manually (whatever you are using is not working)

Reply:
Gotcha, idk! Refactoring of the rest of the file makes sense to come later. Nothing to do here.

Reply:
I believe clang-format does not work with mlir tablegen files. It had some tablegen support but not for MLIR one.